src/billing/components/PayAsYouGo/PayAsYouGo.tsx

```diff
@@ -17,35 +19,45 @@ import CancellationPanel from 'src/billing/components/PayAsYouGo/CancellationPan
 import NotificationPanel from 'src/billing/components/PayAsYouGo/NotificationPanel'
 import InvoiceLoadingWrapper from 'src/billing/components/AssetLoading/InvoiceWrapper'
 import BillingInfoWrapper from 'src/billing/components/AssetLoading/BillingInfoWrapper'
+import {PricingAlert} from 'src/billing/components/PayAsYouGo/PricingAlert'
+
+// Utils
+import {selectCurrentIdentity} from 'src/identity/selectors'
+
+const BillingPayAsYouGo: FC = () => {
+  const {account} = useSelector(selectCurrentIdentity)
+  const directSignup = account.billingProvider === 'zuora'
```

Comment:
```suggestion
  const isDirectSignup = account.billingProvider === 'zuora'
```

Just a suggestion on naming. Since the data type is boolean, adding an `is` makes it easier to read later in the code. Readers don't need to go up to where this variable is defined and then think about what type it is. Just a suggestion, not a blocker. The current code is also good 👍

Reply:
changed